catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -336,4 +341,24 @@ private Metacard getMetacardIfExistsOrNull(String id)
   private interface CheckedSupplier<T> {
     T get() throws Exception;
   }
+
+  private boolean isFormJSONValid(FormTemplate formTemplate) {
+    String jsonString = GSON.toJson(formTemplate);
+    if (isJSONValid(jsonString)) {
+      return true;
+    }
+    LOGGER.warn("Filtered out invalid JSON");
```

Comment:
we should add unit tests for the new validation functionality

Reply:
❓ Doesn't `LOGGER.warn` notify an admin?